<!-- Team:Genspace/Collaborations -->
<!-- http://2016.igem.org/Team:Genspace/Collaborations -->
{{Genspace}}
<html>
	<style>
		.colab{
			padding: 40px 20px;
			margin: 0px 0px 40px;
			border-bottom:1px solid #333;
		}
	</style>
	<div class="column full_size">

		<h1>We helped 3 other iGEM teams:</h1>
		<div class="colab">
			<h3 class="c13 c27" id="h.9qt7tlxjlu4v"><span>1. Stony Brook iGEM Team</span></h3><p class="c13"><span class="c15 c8"><a class="c25" href="http://2016.igem.org/Team:Stony_Brook">
			http://2016.igem.org/Team:Stony_Brook</a></span></p><p class="c1"><span class="c37"></span></p><p class="c13"><span class="c16">The Stony Brook iGEM team undertook the task of investigating the survivability of yeast under desiccation, an experiment equivalent to the one our team has conducted on E. Coli. Given the many potential uses that desiccated yeast could offer to iGEM as well as the scientific community as a whole, we’ve agreed to replicate the desiccation experiments performed by Stony Brook in an effort to corroborate their data. Their protocol consisted of exposing yeast to 2, 4, 6, and 8 hours of dehydration in an incubator. 20 microliters of yeast were placed on separate segments of parafilm which were left in an incubator for the aforementioned amounts of time. Afterwards the yeast was rehydrated and plated. Approximately 2 days later the colonies on the plates were counted. The results of our study are shown below:</span></p><p class="c13"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 408.00px;"><img alt="" src="http://2016.igem.org/wiki/images/6/6c/Colab_stony_brook1.png" style="width: 624.00px; height: 408.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 402.67px;"><img alt="" src="http://2016.igem.org/wiki/images/3/37/Colab_stony_brook2_YeastDehyPic.png" style="width: 624.00px; height: 402.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c5"></span></p><p class="c13"><span class="c16">Note: Given that the amount of yeast that would grow after little or no desiccation would surpass the capacity of the plates or prove difficult to count, we performed a 1 to 10,000 dilution of the 0 hrs of desiccation and multiplied the colonies by 10,000 to account for the dilution. A 1 to 100 dilution was performed on the 2 hrs of desiccation and the colonies multiplied by 100. There were no other dilutions performed on the 4-8 hrs of desiccation.</span></p><p class="c1"><span class="c16"></span></p>

		</div>
		<div class="colab">
			<h3 class="c13 c27" id="h.xt74snpm7krv"><span>2. UGA iGEM Team</span></h3><p class="c13"><span class="c15 c8"><a class="c25" href="http://2016.igem.org/Team:UGA-Georgia">http://2016.igem.org/Team:UGA-Georgia</a></span></p><p class="c1"><span class="c8"></span></p><p class="c13"><span class="c16">We helped the UGA-Georgia iGEM Team by characterizing their mCherry-based reporter system</span></p><p class="c1"><span class="c16"></span></p><p class="c13"><span class="c16">For the second year in a row, the Genspace iGEM team helped the University of Georgia iGEM Team by participating in the Archaeal InterLab Study to further characterize the reproducibility of their mCherry reporter system.  This year, samples were formaldehyde-fixed after oxygen exposure and prior to sending, allowing fluorescence readings to be taken directly without overnight growth.  Triplicate reads were taken with excitation at 590nm and emission at 645nm.</span></p><p class="c1"><span class="c16"></span></p><a id="t.c5456f2804cf0ff579f35f3ce8abe7c16ab04977"></a><a id="t.0"></a><table class="c35"><tbody><tr class="c22"><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c17 c16">Sample ID</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c16 c17">Read 1</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c17 c16">Read 2</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c17 c16">Read 3</span></p></td></tr><tr class="c22"><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">MS</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">0</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">5</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">3</span></p></td></tr><tr class="c22"><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">WT</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">16</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">10</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">0</span></p></td></tr><tr class="c22"><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">A</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">12</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">11</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">11</span></p></td></tr><tr class="c22"><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">B</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">17</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">28</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">13</span></p></td></tr><tr class="c22"><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">C</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">148</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">127</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c6"><span class="c11">123</span></p></td></tr></tbody></table><p class="c1"><span class="c16"></span></p><p class="c12"><span class="c2 c14">Important: As soon as you receive the samples, keep them covered in foil. Exposure to light affects the fluorescence measurement. </span></p><p class="c1 c24"><span class="c2"></span></p><p class="c12" id="h.gjdgxs"><span class="c2">You have been given 5 samples: 3 different mutants (A, B, C), a wild type (WT) and a sample of suspension buffer (MS).  Each tube contains 1mL of the sample; triplicate readings of each sample should be taken. Therefore, 1mL should be enough to run each sample 3 times. </span></p><p class="c1 c24"><span class="c2"></span></p><p class="c12"><span class="c2">*Make sure to vortex tubes before beginning experiment</span></p><p class="c1 c24"><span class="c2"></span></p><p class="c12"><span class="c2 c14">Plate Reader Instructions:</span></p><ol class="c10 lst-kix_5wn82wzao99n-0 start" start="1"><li class="c12 c28"><span class="c2">Using sample “C” determine the linear range for your measurement device.</span></li></ol><ol class="c10 lst-kix_5wn82wzao99n-1 start" start="1"><li class="c12 c18"><span class="c2">We recorded the highest fluorescence from sample “C,” so if this is found to be in the linear range, then all the other samples will be as well. </span></li><li class="c12 c18"><span class="c2">Our device required a 200µL sample, and directly dispensing 200µL from the 1mL sample was within the linear range. No dilution was necessary.</span></li></ol><ol class="c10 lst-kix_5wn82wzao99n-0" start="2"><li class="c12 c28"><span class="c2">Our measurement device is a BioTek Synergy HT plate reader (96 well) and we use the Gen5 software. </span></li></ol><ol class="c10 lst-kix_5wn82wzao99n-1 start" start="1"><li class="c12 c18"><span class="c2">Excitation wavelength (nm): 590/20</span></li><li class="c12 c18"><span class="c2">Emission wavelength (nm): 645/40</span></li><li class="c12 c18"><span class="c2">Scaled* to High Wells</span></li><li class="c12 c18"><span class="c2">Set point Temperature: 25 </span><span class="c2 c21">O</span><span class="c2">C</span></li></ol><p class="c1 c24"><span class="c2"></span></p><p class="c12"><span class="c2">*Your measurement device and protocol may vary from ours and other teams, so please update our form with your specific parameters. Email Ghazal Motakef at </span><span class="c26 c2"><a class="c25" href="mailto:gm76967@uga.edu">gm76967@uga.edu</a></span><span class="c2"> and Anu Romesh at </span><span class="c2 c26"><a class="c25" href="mailto:ar74782@uga.edu">ar74782@uga.edu</a></span><span class="c2"> upon receiving this shipment for the link to the registration form and data form or for any additional questions.</span></p><p class="c1"><span class="c8"></span></p><p class="c1"><span class="c8"></span></p>
		</div>
		<div class="colab">
			<h3 class="c13 c27" id="h.kwlp6bhfze4w"><span>3. BioBricks Foundation iGEM Team</span></h3><p class="c13"><span class="c15 c8"><a class="c25" href="http://2016.igem.org/Team:BioBricks">http://2016.igem.org/Team:BioBricks</a></span></p><p class="c1"><span class="c8"></span></p><p class="c13"><span class="c16">This year, the BioBricks team worked on a functional standard of gene expression and developed an assay to determine whether a gene of interest would have compatibility issues with any module being inserted into the system.  Genspace ran this assay on samples provided by the BioBricks team to test its reproducibility in different labs.</span></p><p class="c1"><span class="c16"></span></p><p class="c12"><span class="c8 c33">Materials Provided</span><span class="c4">:</span></p><ol class="c10 lst-kix_g28xze34k6wo-0 start" start="1"><li class="c7"><span class="c4">7 stab cultures of DH5</span><span class="c3 c14">α</span><span class="c3">Z1 w/ constructs…</span></li></ol><ul class="c10 lst-kix_g28xze34k6wo-1 start"><li class="c12 c20"><span class="c4">“White”</span></li><li class="c12 c20"><span class="c4">“Red”</span></li><li class="c12 c20"><span class="c4">“Cyan”</span></li><li class="c12 c20"><span class="c4">“CFP-RFP 15%”</span></li><li class="c12 c20"><span class="c4">“CFP-RFP 40%”</span></li><li class="c12 c20"><span class="c4">“CFP-RFP 80%”</span></li><li class="c12 c20"><span class="c4">“RFP-CFP”</span></li></ul><p class="c1 c24"><span class="c4"></span></p><p class="c12"><span class="c8 c33">Materials/Equipment Needed</span><span class="c4">:</span></p><ol class="c10 lst-kix_vz7uquou9jas-0 start" start="1"><li class="c7"><span class="c4">LB + Chloramphenicol</span></li><li class="c7"><span class="c4">LB + Chloramphenicol Agar Plates</span></li><li class="c7"><span class="c4">Plate Reader</span></li></ol><ul class="c10 lst-kix_vz7uquou9jas-1 start"><li class="c12 c20"><span class="c8 c33">OD700</span><span class="c4"> (</span><span class="c4 c23">not</span><span class="c4"> OD600 to avoid overlapping with the 610nm emission from mCherry)</span></li><li class="c12 c20"><span class="c4">excitation/emissions - 434/474nm (mTurquoise2) and 587/610nm (mCherry)</span></li></ul><ol class="c10 lst-kix_vz7uquou9jas-0" start="4"><li class="c7"><span class="c4">96 well plate w/ AeraSeal</span></li><li class="c7"><span class="c4">Shaking incubator that works on 96 well plate</span></li></ol><p class="c1 c24"><span class="c4"></span></p><p class="c12"><span class="c8 c33">Protocol:</span></p><ol class="c10 lst-kix_hpwg3j2lv2ms-0 start" start="1"><li class="c7"><span class="c4">Plate Samples from stab cultures on LB + Chloramphenicol plates</span></li><li class="c7"><span class="c4">Pick a colony and start an overnight culture with LB + Chloramphenicol for each construct</span></li><li class="c7"><span class="c4">Back-dilute each @ 1:50 for a final volume of at least 3mL</span></li><li class="c7"><span class="c4">With each culture, Seed an entire row with 200µL/well (12 replicates/sample)</span></li></ol><ul class="c10 lst-kix_hpwg3j2lv2ms-1 start"><li class="c12 c20"><span class="c4">With LB + CAM as a blank in the first row</span></li></ul><ol class="c10 lst-kix_hpwg3j2lv2ms-0" start="5"><li class="c7"><span class="c4">Measure OD700, Fluorescence 434/474 and 587/610</span></li><li class="c7"><span class="c4">Cover with AeraSeal and place in shaking incubator.</span></li><li class="c7"><span class="c4">Take measurements hourly until all OD700’s have stabilized (~10 hours)</span></li></ol><p class="c1 c24"><span class="c4"></span></p><p class="c1 c24"><span class="c4"></span></p><ul class="c10 lst-kix_6li6j6jg1fw4-0 start"></ul>

			<img src="http://2016.igem.org/wiki/images/1/19/Colab-table.png" alt="">
			</div>
		</div>
	</div>
</html>