<!-- http://2016.igem.org/Team:Genspace/Notebook -->
{{Genspace}}
<html>
	<style>
		.day{
			margin: 10px;
			padding: 5px 15px;
		}

		.week{
			margin: 10px 0px;
			padding-bottom:30px;
			border-bottom:1px solid #333;
		}
		
		.week h3, .week h4{
			padding-right: 0px;
		}
	</style>

	<h1>Genspace iGEM notebook page</h1>
	
	<div class="week column full_size">
		<h1 class="c0" id="h.uybqmqvlcwy1"><span>Week 1: June 26 - July 2</span></h1><h3 class="c0" id="h.xzjpmwmkim"><span>6/27/16: </span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Introduction to bacterial transformation and 3A Assembly (Lab safety training) </span></p><ul class="c3 lst-kix_stwlxmi3d6eo-0 start"><li class="c2"><span class="c1">Transformed E.coli with RFP insert </span></li><li class="c2"><span class="c1">Short lesson on 3A Assembly plus first time hands on work </span></li><li class="c2"><span class="c1">Given three parts: TT10 (upstream), 11NB0015 (downstream), pSB1K3 (backbone)</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> New iGem members were able to learn how to conduct transformation according to protocol given and also learned something about 3A Assembly. </span></p><h3 class="c0" id="h.5q6msw3303dv"><span>6/28/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">To learn and go over different useful lab techniques and protocols </span></p><ul class="c3 lst-kix_l17kvfq5p4s3-0 start"><li class="c2"><span class="c1">Todo list: Inspect plates from yesterday&#39;s transformation, colony selection + colony PCR, gel electrophoresis, and start overnight from colony PCR</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">High school students that are new to the team learned about the different protocols and techniques that will be helpful throughout the iGem project. </span></p><p class="c5">

		<img src="http://parts.igem.org/wiki/images/thumb/2/23/Genspace-2016-Notebook-6_28_16.jpg/800px-Genspace-2016-Notebook-6_28_16.jpg" alt=""> 

		</p><h3 class="c0" id="h.pvefftuxt4kx"><span>6/29/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">To learn the mechanism and construct of 3A Assembly and digests of the three parts. </span></p><ul class="c3 lst-kix_9bgep9d2bd7g-0 start"><li class="c2"><span class="c1">Sat down and had a lesson about 3A Assembly </span></li><li class="c2"><span class="c1">High school students learned the mechanism and the construct behind it </span></li><li class="c2"><span class="c1">Performed digests on the three given parts: RFP (upstream), GFP ( downstream), pSB1A3 (backbone)</span></li><li class="c2"><span class="c1">Since the upstream is RFP and downstream GFP the colonies after the 3A Assembly should be orange looking</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> New members now has a better idea of what 3A Assembly is about and starting to understand the basics of the iGem project.</span></p><h3 class="c0" id="h.533chs72ole"><span>6/30/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">To finish and continue the 3A Assembly from yesterday ( ligation + transformation)</span></p><ul class="c3 lst-kix_350o9bwnzj7e-0 start"><li class="c2"><span class="c1">Introduced to ligation and the mechanism behind it</span></li><li class="c2"><span class="c1">Transformed ligated samples int</span><span class="c1">o competent Top10 cells</span></li><li class="c2"><span class="c1">Executed the transformation protocol </span></li><li class="c2"><span class="c1">Plated the samples</span></li><li class="c2"><span class="c1">Learned how to properly label plates and samples</span></li><li class="c2"><span class="c1">Come back tomorrow to check on the plates</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Team members successfully learned how to do a 3A Assembly as well as bacterial transformation. </span></p><h3 class="c0" id="h.721h5rf121x8"><span>7/1/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Another lesson on </span><span class="c1">3A Assembly and learned about the Qubit Fluorometer </span></p><ul class="c3 lst-kix_xyj480yonu7l-0 start"><li class="c2"><span class="c1">Short lesson on 3A Assembly </span></li><li class="c2"><span class="c1">Learned how to use Qubit to read DNA concentrations </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Refreshed members mind about 3A Assembly and learned about a new technique on reading DNA concentrations. </span></p><p class="c5"></p>
		<img src="http://2016.igem.org/wiki/images/thumb/d/d2/Genspace-2016-Notebook-7_1_16.jpg/337px-Genspace-2016-Notebook-7_1_16.jpg" alt="">
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.3ac05rtsik23"><span>Week 2: July 3 - July 9 </span></h1><p class="c5 c7"><span class="c1 c4"></span></p><h3 class="c0" id="h.9kbpzkycjjte"><span>7/5/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Desiccation control of E. Coli</span></p><ul class="c3 lst-kix_k1int1ec5hcv-0 start"><li class="c2"><span class="c1">Introduced to dehydration protocol, given by mentor Mike Flanagan</span></li><li class="c2"><span class="c1">Introduced to serial dilution protocol</span></li><li class="c2"><span class="c1">Verified genes/sequences for synthesis and tardigrade-specific promoters (night meeting)</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Conducted our very first desiccation experiment of E.coli. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/9/9a/Genspace-2016-Notebook-7_5_16%282%29.jpg/450px-Genspace-2016-Notebook-7_5_16%282%29.jpg" alt="">

		</p><h3 class="c0" id="h.g5ihn32bjtzc"><span>7/7/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">To go over the iGEM project/proposal </span></p><ul class="c3 lst-kix_mq0af7su03b7-0 start"><li class="c2"><span class="c1">Listed and discussed the components of our project as a refresher</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Everyone now has an idea of what the project is about and different parts to the project can now be planned out and carry on. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.uvdc06isc7py"><span>Week 3: July 10 - July 16 </span></h1><h3 class="c0" id="h.o8kxqecbzwd"><span>7/11/16:</span><span class="c4"> </span></h3><p class="c5"><span class="c1 c4">Objective:  </span><span class="c1">Learning to and inspecting plates</span></p><ul class="c3 lst-kix_8j9ytyt9pklk-0 start"><li class="c2"><span class="c1">Discussed completed tasks from the previous week</span></li><li class="c2"><span class="c1">Introduced to ampicillin aliquot protocol</span></li><li class="c2"><span class="c1">Made more LB</span></li><li class="c2"><span class="c1">Herded tardigrades</span></li><li class="c2"><span class="c1">Started overnight, will perform serial dilutions tomorrow</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Learned to herd tardigrades for the first time, something that will be crucial for our later tardigrade tests, and everyone understands the protocols for the more common laboratory activities.</span></p><h3 class="c0" id="h.w62f2nybabwn"><span>7/12/16:</span><span class="c4"> </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Working on serial dilutions placed in overnight</span><span class="c1 c4"> </span></p><ul class="c3 lst-kix_8mjmwgch8rxd-0 start"><li class="c2"><span class="c1">Decided on LEA proteins, how to design gRNAs, and began looking for desiccation viability tests for E. coli (night meeting)</span></li><li class="c2"><span class="c1">Girls and guys given individual tasks to elucidate our project</span></li><li class="c2"><span class="c1">Resuspended bacterial colonies to determine survivorship rates after set dehydration time</span></li><li class="c2"><span class="c1">Discussed possible contamination issues</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Getting a head start and went over the desiccation experiment. </span></p><p class="c5 c7"><span class="c1 c4"></span></p><h3 class="c0" id="h.xy3yb08tz3se"><span>7/14/16 </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Counting colonies and come up with a survivability curve for the data</span></p><ul class="c3 lst-kix_q3jzm2sisnf0-0 start"><li class="c2"><span class="c1">Looked over colonies and created semi-log plot to determine survivorship rates</span></li><li class="c2"><span class="c1">Tested tardigrade negative phototaxis</span></li><li class="c2"><span class="c1">Made competent cells</span></li><li class="c2"><span class="c1">Introduced to YPD Agar, produced YPD Agar</span></li><li class="c2"><span class="c1">Wrangled tardigrades</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The desiccation experiment went well and data was collected from each plate. The data then transformed into a graph. Throughout the day more tasks were accomplished and this helps saving time.</span></p><h3 class="c0" id="h.s75pzope5plj"><span>7/15/16 </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Tardigrade work</span></p><ul class="c3 lst-kix_io8hb5sj92b2-0 start"><li class="c2"><span class="c1">Made LB </span></li><li class="c2"><span class="c1">Introduced to yeast nutrient and yeast medium protocols, executed protocols</span></li><li class="c2"><span class="c1">Made Chlorella medium - tardigrade food</span></li><li class="c2"><span class="c1">Poured plates</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">We learned how to culture tardigrades and prepare Chlorella medium.</span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.kl9mcxisnd3p"><span>Week 4: July 17 - July 23</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.4u3k16f8mnaz"><span>7/18/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Transforming E. coli with fluorescent proteins (RFP + GFP) and given weekly schedule</span></p><ul class="c3 lst-kix_jc5eicqtsiwa-0 start"><li class="c2"><span class="c1">Transformed E. coli with RFP and GFP</span></li><li class="c2"><span class="c1">Executed two different wait time on ice for transformation </span></li><li class="c2"><span class="c1">Some people left cells on ice for 30 minutes and others 5 minutes </span></li><li class="c2"><span class="c1">Given schedule of the week from mentor </span></li><li class="c2"><span class="c1">Monday: transformation of RFP + GFP</span></li><li class="c2"><span class="c1">Tuesday: check colonies and PCR out parts </span></li><li class="c2"><span class="c1">Wednesday: PCR clean ups, digest, and ligations </span></li><li class="c2"><span class="c1">Thursday: transformation from ligation </span></li><li class="c2"><span class="c1">Friday: view plates and colonies </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">When using ampicillin backbone, the wait time on ice can be shortened. It does not impact the results for transformation in any significant ways. Thus it saves a lot of time and is still efficient. The given week schedule allow team members to plan ahead and also review what to do everyday. </span></p><p class="c5">


		<img src="http://2016.igem.org/wiki/images/thumb/d/dc/Genspace-2016-Notebook-7_18_16.jpg/450px-Genspace-2016-Notebook-7_18_16.jpg" alt="">

		</p><h3 class="c0" id="h.ugams39fd66d"><span>7/19/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Colony PCR of transformation and second desiccation experiment</span></p><ul class="c3 lst-kix_f0rl8lw43r4y-0 start"><li class="c2"><span class="c1">View plates from yesterday </span></li></ul><ul class="c3 lst-kix_g94mrt9g5p2v-0 start"><li class="c2"><span class="c1">Colony PCR of selected colonies from the transformation</span></li><li class="c2"><span class="c1">Started a second E.coli desiccation experiment </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Plates from yesterday came out well and colony PCR was done. Started our second desiccation experiment hoping the results to be almost the same as the first experiment. Aiming for consistency. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.gguvy651em8z"><span>7/20/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Qubit reading of plasmids from last week, tardigrades wrangling, and colony PCR for the CRISPR workshop </span></p><ul class="c3 lst-kix_yzhee0rcswca-0 start"><li class="c2"><span class="c1">Helped colony PCR samples for the CRISPR workshop</span></li><li class="c2"><span class="c1">Qubit reading of last week’s plasmids from the mini preps </span></li><li class="c2"><span class="c1">Made and poured more LB plates </span></li><li class="c2"><span class="c1">Tardigrade wrangling </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Have more experience on how to use the Qubit and also learned how to isolate tardigrades. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.af5dj6kpesvc"><span>7/21/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Colony counting from desiccation experiment and 3A Assembly of RFP and GFP</span></p><ul class="c3 lst-kix_5jznhf7l91c0-0 start"><li class="c2"><span class="c1">Ligated parts from digests on 7/18/16</span></li><li class="c2"><span class="c1">Counted colonies on plates from the desiccation experiment conducted on 7/19/16</span></li><li class="c2"><span class="c1">Transformation was done after the ligation </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Ligated parts of the digests and continued with the 3A Assembly. The transformation steps went well and the plates will be inspected tomorrow. The colonies from the desiccation experiment was pretty consistent with the first. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.9zkg2rmv0825"><span>Week 5: July 24 - July 30</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.h845qw87dl5v"><span>7/25/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Troubleshooting issues with GFP and RFP transformation</span></p><ul class="c3 lst-kix_zd3uyfwagglq-0 start"><li class="c2"><span class="c1">The plates did not come out the way we wanted it to be. So therefore an investigation of what we did for the past week was listed</span></li></ul><ul class="c3 lst-kix_c3obq3u0fy1h-0 start"><li class="c2"><span class="c1">Day 1: Transforming E.coli - positive control worked therefore the transformation should work </span></li><li class="c2"><span class="c1">Day 2: Colony selection and Colony PCR - might have selected wrong colonies to work with </span></li><li class="c2"><span class="c1">Day 3: PCR clean ups - found foreign DNA inside samples after the cleanup was done </span></li><li class="c2"><span class="c1">Day4: Transformation using legations - possibly wrong backbone/ not enough backbone or possible error with ligation </span></li><li class="c2"><span class="c1">Used old plates and re selected colonies for Colony PCR</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The transformation from yesterday did not turn out the way it&#39;s suppose to be. Therefore we came up possible errors that may occurred during different days. As a result we re conducted the experiment from Colony selection hoping that it would work this time. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/4/43/Genspace-2016-Notebook-7_25_16.jpg/800px-Genspace-2016-Notebook-7_25_16.jpg" alt="">

		</p><h3 class="c0" id="h.eybl1osvcot5"><span>7/26/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Continuing to troubleshoot</span></p><ul class="c3 lst-kix_43lilecf5kkh-0 start"><li class="c2"><span class="c1">PCR cleanup the RFP and GFP</span></li><li class="c2"><span class="c1">Made 0.7% agarose gel for viewing the base pairs of the plasmid mini preps </span></li><li class="c2"><span class="c1">Gel results weren’t great, most likely due to using incorrect pipette size</span></li><li class="c2"><span class="c1">Plasmid mini prep of the GFP and RFP</span></li><li class="c2"><span class="c1">Qubit readings of the RFP and GFP after the plasmid mini prep </span></li><li class="c2"><span class="c1">Digested upstream (RFP), downstream (GFP), and backbone (pSB1A3)</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Everyone on the team ended up redoing all the steps from past week. We are hoping that we can identify the problem through this rerun. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.chx9kvf1ucch"><span>7/27/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Finishing troubleshooting</span></p><ul class="c3 lst-kix_atus64bn76c8-0 start"><li class="c2"><span class="c1">Worked with remaining components of 3A assembly (ligation + transformation)</span></li><li class="c2"><span class="c1">Produced 0.7% agarose gel to see results of digest</span></li><li class="c2"><span class="c1">Replicated linearized plasmid backbones using PCR</span></li><li class="c2"><span class="c1">Began collaborating with Stony Brook University</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Last day of troubleshooting our mistakes. Hope everything will turn out the way we wanted tomorrow. The plates will be viewed tomorrow as well. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.lstby9e7vvqg"><span>7/28/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Checking transformed cells, completing linearized backbone, and making yeast media </span></p><ul class="c3 lst-kix_75cc7ye40zsj-0 start"><li class="c2"><span class="c1">NEB 2.1 digests of the backbones (1A3, 1C3, and 1K3)</span></li><li class="c2"><span class="c1">PCR cleanup</span></li><li class="c2"><span class="c1">Reviewing transformed cells </span></li></ul><ul class="c3 lst-kix_75cc7ye40zsj-1 start"><li class="c5 c10"><span class="c1">Positive worked, negative worked, other colonies exist in other plates but it&#39;s taking awhile for those colonies to take in the fluorescence </span></li></ul><ul class="c3 lst-kix_75cc7ye40zsj-0"><li class="c2"><span class="c1">Made yeast media</span></li><li class="c2"><span class="c1">Discussed Stony Brook collaboration</span></li><li class="c2"><span class="c1">Gel was ran with the digests of the backbones</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The plates came out to be ok. It took the colonies a while to turn red. On the good side the positive and negative controls worked. The gel for chloramphenicol and kanamycin came out good. But the ampicillin did not have the correct base pairs. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.guaaw2b0qn29"><span>7/30/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Receiving and working with materials from Stony Brook</span></p><ul class="c3 lst-kix_kwx6nkxxwwd-0 start"><li class="c2"><span class="c1">Transformed plasmid into competent cells (top10) </span></li><li class="c2"><span class="c1">Gave Stony Brook materials for our experiment for replication of our experiment</span></li></ul><ul class="c3 lst-kix_kwx6nkxxwwd-1 start"><li class="c5 c10"><span class="c1">Confirming they have same results</span></li></ul><ul class="c3 lst-kix_kwx6nkxxwwd-0"><li class="c2"><span class="c1">Began liquid culture of Stony Brook yeast (BY4741) </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> First collaboration with a different iGem team. This fulfills one of the gold medal requirements. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.2rko1i673dr1"><span>Week 6: July 31 - August 6 </span></h1><h3 class="c0" id="h.d87nexjt64sw"><span>7/31/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Continuing to work with Stony Brook materials</span></p><ul class="c3 lst-kix_z8kexfgxtzw4-0 start"><li class="c2"><span class="c1">Created 4 aliquots</span></li><li class="c2"><span class="c1">Took 1 colony that had been transformed with plasmid and turned into liquid culture yesterday</span></li></ul><ul class="c3 lst-kix_z8kexfgxtzw4-1 start"><li class="c5 c10"><span class="c1">Result is significant amount of colonies, possibly not right amount</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> First collaboration with a different iGem team. This fulfills one of the gold medal requirements.</span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.du6i2j9khj5m"><span>8/1/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Cloning Biobricks, Re PCR-ing linearized backbones (ampicillin only), performing colony PCR on plates from 7/27/16, and making yeast competent cells </span></p><ul class="c3 lst-kix_bb9ottp7csh4-0 start"><li class="c2"><span class="c1">Transforming Stony Brook yeast into competent cells</span></li><li class="c2"><span class="c1">Received Zymo Prep Kit competent cell and transformation protocol</span></li><li class="c2"><span class="c1">Performed colony PCR on plates from 7/27</span></li></ul><ul class="c3 lst-kix_bb9ottp7csh4-1 start"><li class="c5 c10"><span class="c1">Result lacked orange color, thus performing another colony PCR</span></li></ul><ul class="c3 lst-kix_bb9ottp7csh4-0"><li class="c2"><span class="c1">PCR’d linearized pSB1A3 plasmid backbone</span></li><li class="c2"><span class="c1">Introduced to g-blocks and cloned RFP, AAVLEA3, and pSB1C3 backbone </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Received g blocks from IDT and we can now start the iGem project by re suspending those g blocks. The ampicillin backbone from the other day did not turn out great so therefore we re linearized the backbone. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/8/81/Genspace-2016-Notebook-8_1_16.jpg/450px-Genspace-2016-Notebook-8_1_16.jpg" alt="">

		</p><h3 class="c0" id="h.3w797ixmiov2"><span>8/2/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Make more competent cells and also run a gel </span></p><ul class="c3 lst-kix_1vfgrzmwgm1c-0 start"><li class="c2"><span class="c1">Competent cells were made and stored in -80C </span></li><li class="c2"><span class="c1">Planned ahead for human practices event ( outreach program at the NY Hall of Science)</span></li><li class="c2"><span class="c1">Replicated 1A3 plasmids and ran a gel</span></li></ul><ul class="c3 lst-kix_1vfgrzmwgm1c-1 start"><li class="c5 c10"><span class="c1">Result shows contamination because of presence of genomic DNA</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The 1A3 backbone did not work again but luckily we found out the reason why it wasn&#39;t working. The freshly made competent cells can now allow us to conduct more bacterial transformation in the next month or so. </span></p><p class="c5">


		<img src="http://2016.igem.org/wiki/images/thumb/3/31/Genspace-2016-Notebook-8_2_16.jpg/450px-Genspace-2016-Notebook-8_2_16.jpg" alt="">
		

		</p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.i0k7p8hw4pex"><span>8/3/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">R</span><span class="c1">estriction digest and ligation, make/pour plates, and creating pSB1A3</span></p><ul class="c3 lst-kix_iypat5fw897s-0 start"><li class="c2"><span class="c1">Performed restriction digests and ligations</span></li><li class="c2"><span class="c1">Poured more plates for future experiments </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Digested and ligated the g blocks we received the other day and transformation of the ligation will take place tomorrow. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/5/5e/Genspace-2016-Notebook-8_3_16%281%29.jpg/450px-Genspace-2016-Notebook-8_3_16%281%29.jpg" alt="">

		</p><h3 class="c0" id="h.qu22cokqvwgq"><span>8/4/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Transforming g-blocks and amplifying backbones</span></p><ul class="c3 lst-kix_bcw736a9swox-0 start"><li class="c2"><span class="c1">Transformed and plated ligations from yesterday</span></li><li class="c2"><span class="c1">Resuspended and performed restriction digest of latest DNA synthesis</span></li><li class="c2"><span class="c1">PCR linearization of pSB3K3 and pSB4A3</span></li><li class="c2"><span class="c1">Running gel with PCR linearization of pSB1A3</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The gel came out well for pSB1A3 and we started the linearization for two other backbones. The transformation of the ligation from yesterday was conducted and the plates will be checked on tomorrow morning. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.k8eyqk2yr9nk"><span>Week 7: August 7 - August 13</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.gjbbcxe9i417"><span>8/7/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> PCR of ligations and also running a gel </span></p><ul class="c3 lst-kix_4gskswi3ikoc-0 start"><li class="c2"><span class="c1">Selected 5 colonies from each plate (plates consisted of either: LEA and backbone; RFP and backbone; or LEA and backbone) </span></li><li class="c2"><span class="c1">A gel was ran for the selected colonies </span></li><li class="c2"><span class="c1">Started overnights for the colonies that were good </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The gel came out nice for all the samples. Roughly all of the colonies has the correct number of base pairs. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.6cacbuwiq62r"><span>8/8/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Mini Prep of the overnights and Qubit</span><span class="c1">, continuing 3A assembly (ligation + transformation), and tardigrade wrangling</span></p><ul class="c3 lst-kix_6pgpgg4audfz-0 start"><li class="c2"><span class="c1">Waking up promoters and ribosome binding sites from iGEM registry</span></li></ul><ul class="c3 lst-kix_6pgpgg4audfz-1 start"><li class="c5 c10"><span class="c1">Woke up J0500, B0034, and J04500</span></li></ul><ul class="c3 lst-kix_6pgpgg4audfz-0"><li class="c2"><span class="c1">Ligation (GFP) and digest (HDLEA3)</span></li><li class="c2"><span class="c1">Transformation of GFP ligation</span></li><li class="c2"><span class="c1">Tardigrade wrangling</span></li><li class="c2"><span class="c1">Mini prepped LEA proteins and RFP cultures</span></li><li class="c2"><span class="c1">Created glycerol stocks of LEA proteins and RFP</span></li><li class="c2"><span class="c1">Performed qubit </span></li><li class="c2"><span class="c1">Prepared mini prepped plasmid DNA for sequencing</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The finished 3A Assembly samples are now transformated and mini prepped. We can now prepare those samples and have them ready for sequencing. </span></p><p class="c5 c7"><span class="c1 c4"></span></p><h3 class="c0" id="h.9ofsg2d0m87n"><span>8/9/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Continuing 3A assembly, performing colony PCR, making plates, and tardigrade wrangling</span></p><ul class="c3 lst-kix_ng9g14ceywh4-0 start"><li class="c2"><span class="c1">Resuspended and digested new DNA (HDLEA2, MAHS)</span></li><li class="c2"><span class="c1">Ligated and transformed yesterday’s digest (HDLEA3)</span></li><li class="c2"><span class="c1">Miniprepped overnights from yesterday (RFP, AAVLEA1, RvLEA)</span></li><li class="c2"><span class="c1">Performed colony PCR on yesterday’s transformations </span></li><li class="c2"><span class="c1">Made Chlor/LB and SC Leu- plates</span></li><li class="c2"><span class="c1">Made LB</span></li><li class="c2"><span class="c1">Wrangled tardigrades</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">We receive new g blocks today and resuspended them. We finished the ligation from yesterday&#39;s transformation and conducted Colony PCR of those samples. During remaining of the time fresh plates were made for the transformation. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/e/e4/Genspace-2016-Notebook-8_9_16.jpg/450px-Genspace-2016-Notebook-8_9_16.jpg" alt="">

		</p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.qsacjj321voi"><span>8/10/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Performing colony PCR, performing ligation, and analyze gels</span></p><ul class="c3 lst-kix_7086d2u8oim5-0 start"><li class="c2"><span class="c1">Performed colony PCR of GFP 1-5 and HDLEA3 </span></li><li class="c2"><span class="c1">Ligation of HDLEA2 and MAHS</span></li><li class="c2"><span class="c1">Miniprepped I0500 and qubit analysis </span></li><li class="c2"><span class="c1">Analyzed plasmid products on gels (AAVLEA3, RFP-071816, RvLEA) </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Since the g blocks of different samples came in different times, the steps we’re up to for each sample is different. We continued and picked up from where we left off yesterday. </span></p><h3 class="c0" id="h.lljmffv9jy1e"><span>8/11/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> PCRing plasmid products and transforming g-blocks</span></p><ul class="c3 lst-kix_3qiec81lnmc1-0 start"><li class="c2"><span class="c1">PCR’d out insert of all plasmid products</span></li></ul><ul class="c3 lst-kix_3qiec81lnmc1-1 start"><li class="c5 c10"><span class="c1">AAVLEA3, RFP-071816, RvLEA</span></li></ul><ul class="c3 lst-kix_3qiec81lnmc1-0"><li class="c2"><span class="c1">Transformed ligated HDLEA2 and MAHS and plated</span></li><li class="c2"><span class="c1">Ran 1.0% agarose gel to analyze PCR product of GFP and HDLEA3</span></li><li class="c2"><span class="c1">Made overnight of colony resuspension from yesterday 8/10</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Were able to analyze the result of the PCR of GFP and HDLEA3 </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/a/a1/Genspace-2016-Notebook-8_11_16.jpg/800px-Genspace-2016-Notebook-8_11_16.jpg" alt="">

		</p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.l2odxd5u44fz"><span>Week 8: August 14 - August 20</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.ta69yv7wkwh9"><span>8/15/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Split into teams and worked on different parts (Making new construct, performing colony PCR, and waking up parts)</span></p><ul class="c3 lst-kix_m7igreeowcuh-0 start"><li class="c2"><span class="c1">Resuspended new primer (I0500-B0034-Spe1 RVS)</span></li></ul><ul class="c3 lst-kix_m7igreeowcuh-1 start"><li class="c5 c10"><span class="c1">PCR’d the construct</span></li></ul><ul class="c3 lst-kix_m7igreeowcuh-0"><li class="c2"><span class="c1">Colony PCR’d yesterday’s MAHS and HDLEA2 transformations</span></li></ul><ul class="c3 lst-kix_m7igreeowcuh-1 start"><li class="c5 c10"><span class="c1">Started overnight and prepared for sequencing</span></li></ul><ul class="c3 lst-kix_m7igreeowcuh-0"><li class="c2"><span class="c1">Ran AAVLEA3, RFP-07186, and RvLEA3 in gel</span></li></ul><ul class="c3 lst-kix_m7igreeowcuh-1 start"><li class="c5 c10"><span class="c1">Prepared AAVLEA3 and RvLEAM for sequencing</span></li></ul><ul class="c3 lst-kix_m7igreeowcuh-0"><li class="c2"><span class="c1">Miniprepped and qubit’d HDLEA3-1 and HDLEA3-3</span></li><li class="c2"><span class="c1">Woke up parts identified on Thursday and made glycerol stock of J04450 </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> By splitting into teams we were able to get different tasks done at a faster rate. Different experiments and protocols took place at once. Unfortunately, the colonies from colony PCR did not have any hits. So therefore the colony PCR of HDLEA2 and MAHS will be repeated tomorrow. </span></p><p class="c5"><span class="c1">
		
		<img src="http://2016.igem.org/wiki/images/thumb/6/6a/Genspace-2016-Notebook-8_15_16.jpg/450px-Genspace-2016-Notebook-8_15_16.jpg" alt="">

		</span></p><h3 class="c0" id="h.25sn0bah3bvz"><span>8/16/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> To redo colony PCR of HDLEA2 and MAHS. Analyzing primer and linearized plasmid backbones</span></p><ul class="c3 lst-kix_kxlmm7584tpg-0 start"><li class="c2"><span class="c1">Ran 1.0% agarose gel for analyzing I0500 primer and 3K3 and 4A3 linearized backbone</span></li><li class="c2"><span class="c1">Colony PCR’d 5 colonies from each plate of MAHS and HDLEA2 </span></li><li class="c2"><span class="c1">Performed colony selection for overnight of promoters </span></li><li class="c2"><span class="c1">Performed Dpn1 digest and Qubit of linearized backbones</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Unlike yesterday, we were able to get hit on for the MAHS and also HDLEA2. An overnight of both samples were prepared and by tomorrow the overnight will be made into glycerol stock and mini prepped. </span></p><p class="c5"><span class="c1">
		
		<img src="http://2016.igem.org/wiki/images/thumb/5/53/Genspace-2016-Notebook-8_16_16%281%29.jpg/450px-Genspace-2016-Notebook-8_16_16%281%29.jpg" alt="">

		</span></p><h3 class="c0" id="h.bt3h70dfz8mn"><span>8/17/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Miniprepping and ligating digests</span></p><ul class="c3 lst-kix_6aig5hyheqpx-0 start"><li class="c2"><span class="c1">Miniprepped L1, L2, and L3 cultures from glycerol stocks (and of pSB1C3 cultures)</span></li><li class="c2"><span class="c1">Qubit’d miniprepped plasmid DNA</span></li><li class="c2"><span class="c1">Ran 0.7% gel for HDLEA2, MAHS, RFP, pSB1C3, and L1-L3 miniprep products</span></li><li class="c2"><span class="c1">Discussed LEA proteins to be amplified and sequenced</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Valuable information was gained regarding the miniprep products of the LEA proteins during the gel; crucial information regarding the amplification and sequencing of the LEA proteins was also discussed</span></p><h3 class="c0" id="h.11apwoftl0gy"><span>8/18/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Split into different teams and worked on different parts. </span></p><ul class="c3 lst-kix_i9kivhxivyp-0 start"><li class="c2"><span class="c1">Amplified LEA proteins AAvLEA1, RvLEAM, and HDLEA3 </span></li></ul><ul class="c3 lst-kix_cw9gj8j1hxok-0 start"><li class="c2"><span class="c1">Transformed HDLEA1 ligation</span></li><li class="c2"><span class="c1">Suspended primers VF2 and VR</span></li><li class="c2"><span class="c1">Re Qubit and sequence sample prep</span></li><li class="c2"><span class="c1">Performed PCR cleanup and qubit of I0500 insert into B0034 plasmid (1A3)</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Transformation of the HDLEA1 ligation would be crucial for our later experiments</span></p><p class="c5"><span class="c1">

		<img src="http://2016.igem.org/wiki/images/thumb/4/47/Genspace-2016-Notebook-8_18_16.jpg/800px-Genspace-2016-Notebook-8_18_16.jpg" alt="">

		</span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5"><span class="c9">8/19/16:</span></p><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Running gel with PCR products</span></p><ul class="c3 lst-kix_tb1vg58laamp-0 start"><li class="c2"><span class="c1">Ran gel with HDLEA2, MAHS, and RFP products</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Valuable information regarding the LEA proteins was obtained through the gel </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.mw7we3hgcfab"><span>Week 9: August 21 - August 27</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.gfwvlf6b73qt"><span>8/22/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Performing restriction digests and reviewing sequencing results (was once again split into different teams)</span></p><ul class="c3 lst-kix_2xfatwngxco3-0 start"><li class="c2"><span class="c1">Performed restriction digest of I0500 and pSB1A3 </span></li><li class="c2"><span class="c1">Performed colony selection, PCR, and made overnight of B0034 and HDLEA1 </span></li><li class="c2"><span class="c1">Created overnight of the colony PCR of HDLEA2 and MAHS</span></li><li class="c2"><span class="c1">Reviewed sequencing results from Genewiz for pre-3A assembly</span></li><li class="c2"><span class="c1">Reviewed I0500 and RBS protocol</span></li><li class="c2"><span class="c1">Digest of LEA proteins with chloramphenicol linearized backbone</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1">  Team members were once again split into different teams. Each team had different jobs and worked on different parts. At the same time we were introduced to I0500 and the ribosome binding site protocol. The team is trying to come up with different but new parts to be submitted to the iGem registry. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/b/b5/Genspace-2016-Notebook-8_22_16.jpg/800px-Genspace-2016-Notebook-8_22_16.jpg" alt="">


		</p><h3 class="c0" id="h.bnwlg6c6n5ly"><span>8/23/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Noticed mistakes and redid necessary steps.</span></p><ul class="c3 lst-kix_2uk9smt74a26-0 start"><li class="c2"><span class="c1">Resuspended G blocks and also ligated + transformed</span></li></ul><ul class="c3 lst-kix_hewitj45578-0 start"><li class="c2"><span class="c1">Ran a gel and performed colony PCR</span></li></ul><ul class="c3 lst-kix_amndzgmhvrc9-0 start"><li class="c2"><span class="c1">Ran gel electrophoresis of B0034 and HDLEA1 </span></li></ul><ul class="c3 lst-kix_amndzgmhvrc9-1 start"><li class="c5 c10"><span class="c1">B0034 was okay, HDLEA1 not good</span></li></ul><ul class="c3 lst-kix_amndzgmhvrc9-0"><li class="c2"><span class="c1">Re-performed colony PCR of HDLEA1 samples</span></li><li class="c2"><span class="c1">Started overnight for B0034 samples</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Resuspension, ligation, and transformation of the g-blocks would be important for later DNA submission; determining the efficacy of the HDLEA1 meant we knew what we had to work on</span></p><h3 class="c0" id="h.c7p21x7kpc0b"><span>8/24/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Performing colony PCR, running a gel, miniprepping and qubitting</span></p><ul class="c3 lst-kix_yzet3cwurdjl-0 start"><li class="c2"><span class="c1">Performed colony PCR of I0500, AAvLEA1, RvLEAM, HDLEA3, HDLEA1, and MAHS</span></li><li class="c2"><span class="c1">Ran gel for digest </span></li><li class="c2"><span class="c1">Created glycerol stock of B0034 </span></li></ul><ul class="c3 lst-kix_yzet3cwurdjl-1 start"><li class="c5 c10"><span class="c1">Miniprepped and qubit’d B0034 </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> The colonies we selected for each sample came out to be satisfying. Each of the proteins had at least 1 hit and an overnight was started for each.</span></p><h3 class="c0" id="h.2w1qsnt41pqc"><span>8/25/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Performing colony PCR and miniprepping and qubitting LEA proteins</span></p><ul class="c3 lst-kix_ybbc06ioqnqp-0 start"><li class="c2"><span class="c1">Performed colony PCR of AAvLEA1</span></li><li class="c2"><span class="c1">Miniprepped HDLEA3 and HDLEA1</span></li><li class="c2"><span class="c1">Qubit’d HDLEA1, HDLEA3, AAvLEA1, MAHS, and I0500 + B0034</span></li><li class="c2"><span class="c1">PCR linearization and amplification of HDLEA1, HDLEA3, I0500 + B0034, MAHS, and AAvLEA1 </span></li></ul><ul class="c3 lst-kix_ybbc06ioqnqp-1 start"><li class="c5 c10"><span class="c1">Also created overnights of above constructs for glycerol stocks</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Due to the delay in AAvLEA1, every other proteins is a step ahead. Running in parallel colony PCR and gel analysis of AAvLEA1 came out to be satisfying. Overnights were started and it is sure catching up. The other proteins were qubitted and had low concentrations. PCR amplifications were ran trying to increase the concentration. The amplification came ended up well. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.xt9yt6hk7pzw"><span>Week 10: August 28 - September 3</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.27lioto5bzle"><span>8/28/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Team brunch and tardigrade hunting </span></p><ul class="c3 lst-kix_wjvmst5ofsf9-0 start"><li class="c2"><span class="c1">Ran gel for colony PCR of RvLEAM, PCR products of amplified LEA proteins, and I0500</span></li><li class="c2"><span class="c1">Made glycerol stocks of HDLEA1, HDLEA3, and AAvLEA1</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> The team brunch brought the team members together to discuss the progression of the project and also different team members brought in different moss and algae samples hoping to find some tardigrades in them. After brunch, a gel was tran for the LEA proteins and the gel came out to be fairly well.</span></p><h3 class="c0" id="h.8fydnkmqi9dp"><span>8/29/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Performing Dpn1 digest, PCR cleanups, Qubit analysis, and 3A digest of LEA proteins</span></p><ul class="c3 lst-kix_hk4935c2gpnh-0 start"><li class="c2"><span class="c1">Performed Dpn1 digest and PCR cleanup of amplified products</span></li></ul><ul class="c3 lst-kix_hk4935c2gpnh-1 start"><li class="c5 c10"><span class="c1">Qubit’d products</span></li></ul><ul class="c3 lst-kix_hk4935c2gpnh-0"><li class="c2"><span class="c1">Performed 3A digest of HDLEA1, HDLEA3, AAvLEA1, and J04560 (with K880006 as backbone) </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> The digest of each LEA protein is finished. Tomorrow we will be ligating the parts together hoping the final ligation will give us positive and happy results.</span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/3/39/Genspace-2016-Notebook-8_29_16.jpg/450px-Genspace-2016-Notebook-8_29_16.jpg" alt="">
		
		</p><h3 class="c0" id="h.btdgunwlr3y0"><span>8/30/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">PCR clean up of MAHS and RVLEAM, qubit reading of only RVLEAM, ligation of HDLEA1, HDLEA3, AAVLEA1, AND JO4560</span></p><ul class="c3 lst-kix_5lzg8dcprdp7-0 start"><li class="c2"><span class="c1">Miniprepped RVLEAM (1) and (4)</span></li><li class="c2"><span class="c1">Qubit readings came out low therefore a digest was done and also ran a gel</span></li><li class="c2"><span class="c1">Forgot to make glycerol stock for RVLEAM, therefore a new overnight was made</span></li><li class="c2"><span class="c1">6x ratio ligation of the LEAs</span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> The ligation was successfully finished and we are now ready to transform the LEAs. </span></p><h3 class="c0" id="h.6nqyrydv1m7u"><span>8/31/16:</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Made glycerol stock for MAHS and RVLEAM, plasmid miniprep, Qubit reading, transformation of the ligations of the LEAs from 8/30/16</span></p><ul class="c3 lst-kix_40c1m7ftdxik-0 start"><li class="c2"><span class="c1">Made glycerol stock of MAHS since it went missing. Made glycerol stock for RVLEAM from new overnight made on 8/30/16</span></li><li class="c2"><span class="c1">Plasmid miniprepped the two LEAs</span></li><li class="c2"><span class="c1">Qubit readings came out low, a digest and gel was ran </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Transformation of the LEAs is finished and the plates will be ready to view tomorrow. Hope there will be great news!</span></p><h3 class="c0" id="h.onib3i68tz0y"><span>9/1/16: </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Review plates from the LEAs ligations, colony PCR and gel analyzation of PCR products</span></p><ul class="c3 lst-kix_xdiu0j24r0x5-0 start"><li class="c2"><span class="c1">Transformation of the LEAs came out great but the colonies were green </span></li><li class="c2"><span class="c1">Colony PCR was done and a gel was ran</span></li><li class="c2"><span class="c1">The gel came out fine but there were bands at 3000bp resulting in inconclusive results</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The results of the transformed plates fluorescing green indicates the misusage of the promoter, RBS, and backbone part. The goal was to use the K880005 part, but the green colonies show that this was not the part that was ligated. Colony PCR was conducted to analyze the confusion. This concluded that the wrong vector was used for the insert. Thus initiating a rebuilding of parts. </span></p><p class="c5">


		<img src="http://2016.igem.org/wiki/images/thumb/c/ce/Genspace-2016-Notebook-9_1_16.jpg/800px-Genspace-2016-Notebook-9_1_16.jpg" alt="">
		</p><p class="c5 c7"><span class="c11"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.uupj83r6bmm5"><span>Week 11: September 8 - September 14</span></h1><h3 class="c0" id="h.j0rivdwq1amn"><span>9/8/16:</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Rebuilding whole construct: Restriction Digest and Ligation of PCR products with gel analyzation</span></p><ul class="c3 lst-kix_ypsy5atwyi6o-0 start"><li class="c2"><span class="c1">Restriction digest of MAHS, HDLEA1, HDLEA3, and K880005 from plasmid DNA</span></li><li class="c2"><span class="c1">Gel analyzation of PCR products</span></li><li class="c2"><span class="c1">Ligation of inserts to vector (K880005) </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Digest of plasmid DNA were analyzed through a gel. The parts were the correct size, thus used for ligation into correct vector part. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.vsyit8tcc3kw"><span>9/12/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Bacterial Transformation of ligated parts from 9/8/16. Designed new primers to fix problem on HDLEA2 coding sequence. </span></p><ul class="c3 lst-kix_o5gf7cmtwsor-0 start"><li class="c2"><span class="c1">Bacterial transformation of ligated parts from previous lab day into </span><span class="c1 c12">E. coli. </span></li><li class="c2"><span class="c1">HDLEA2 had an illegal PstI restriction site in middle of its CDS. Primers were designed using Goldengate assembly to target illegal site and change one base pair on codon. </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Problem with HDLEA2 was analyzed and solution was designed. Continuation of fixing the problem with plasmid product from 9/8/16 through bacterial transformation. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.43hsvflugy4l"><span>9/13/16</span></h3><p class="c5"><span class="c1 c4">Objective:</span><span class="c1"> Colony PCR of plates from 9/12/16. </span></p><ul class="c3 lst-kix_sskncqirz1pj-0 start"><li class="c2"><span class="c1">Plates were analyzed and colony PCR was started to see if part was properly built and correct size. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Inspection of transformed colonies on plates. Colony PCR executed to conclude on correct size and part used for ligation. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.kxff6b2xuq9f"><span>9/14/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Gel analyzation of Colony PCR products and Overnights made from colony suspensions</span></p><ul class="c3 lst-kix_iqsroccfh4hx-0 start"><li class="c2"><span class="c1">PCR products from previous day were run on a 0.7% agarose gel to compare base pair size results to those designed. </span></li><li class="c2"><span class="c1">Overnights were made from picking one sample from each LEA protein (HDLEA1 #1, HDLEA3 #2, AAvLEA1 #8, and MAHS #8)</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">One colony suspension sample was picked from each protein from colony PCR. Insert size was compared using gel electrophoresis. These samples will be miniprep for sequencing. </span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.9e21rewmfxbs"><span>Week 12: September 15 - September 21</span></h1><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5"><span class="c9">9/15/16</span></p><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Plasmid miniprep of overnights + preparation of sequencing plasmids.</span></p><ul class="c3 lst-kix_l9d7lhp73k10-0 start"><li class="c2"><span class="c1">Glycerol stock made of each overnight. Flash freezed in liquid nitrogen. </span></li><li class="c2"><span class="c1">Plasmid miniprep of left over overnight after glycerol stock. </span></li><li class="c2"><span class="c1">QuBit analysis of DNA for concentration to prepare for sequencing.</span></li><li class="c2"><span class="c1">Samples each had forward and reverse primer sent with. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Plasmids that were isolated had high enough concentrations to be sent for sanger sequencing to conclude on the correct built part: Promoter + RBS + 1C3 Backbone + Insert from each protein (PR1_’Insert’)</span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.i7zdc6kp7g06"><span>9/19/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Discussion on sequencing results. Focus on one protein to do quality and method control experiment: Restriction digest and ligation of MAHS with bacterial transformation.</span></p><ul class="c3 lst-kix_kqsjweg52eb-0 start"><li class="c2"><span class="c1">Sequencing results came back inconclusive where there were multiple problems among the constructs. We chose to choose one protein to focus on to isolate problems.</span></li><li class="c2"><span class="c1">Restriction digest of MAHS. Used linearized plasmid from 29, August. 2016.</span></li><li class="c2"><span class="c1">Ligation of MAHS insert with PR1_ vector. </span></li><li class="c2"><span class="c1">Bacterial transformation of construct into cells. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Results from sequencing all constructs were inconclusive. Many major problems came up. Chose to focus on one protein to go through the whole process for sequencing to provide an insight to the problems. </span></p><p class="c5"><span class="c1">*Problem with bacterial transformation: LB was not added to cells prior to 2-hour incubation period. Transformation was redone on 9/20/16 </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1 c4"></span></p><h3 class="c0" id="h.1y8gzo47v7ts"><span>9/21/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Colony PCR of plates with gel analyzation.</span></p><ul class="c3 lst-kix_bwx9pbpm1pco-0 start"><li class="c2"><span class="c1">Colonies were picked from plates for colony PCR. </span></li><li class="c2"><span class="c1">PCR products were placed on a 0.7% gel to conclude on correct size. </span></li><li class="c2"><span class="c1">Overnights made of colonies with correct base pairs. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Colony PCR led to two colonies with the correct size. Overnights were made from the colony suspension to be mini prepped for plasmid DNA. </span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.8jh5g8ilw0db"><span>Week 13: September 22 - September 28</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.d45czk6jcbtu"><span>9/22/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Plasmid miniprep and PCR Amplification from overnights. </span></p><ul class="c3 lst-kix_rtwieied52wi-0 start"><li class="c2"><span class="c1">Plasmid miniprep of the two overnights after glycerol stock. </span></li><li class="c2"><span class="c1">QuBit analysis of plasmid DNA. Concentrations were very low. </span></li><li class="c2"><span class="c1">PCR amplification to follow to up DNA concentration before sending to sequencing. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">After plasmid miniprep, the analysis of DNA concentration was low, thus we amplified the plasmid products. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.1ubgjh5vlpul"><span>9/24/16 </span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Start of Microinjection of tardigrades with Cas-9 system. </span></p><ul class="c3 lst-kix_aa5nmsn6256q-0 start"><li class="c2"><span class="c1">Travelled to Stony Brook University to use microinjection machine for other part of project regarding the insertion of fluorescent protein into tardigrades. </span></li><li class="c2"><span class="c1">Tardigrades needed to be isolated and placed on microinjection slide to be injected. </span></li><li class="c2"><span class="c1">The lab was based on microinjection </span><span class="c1 c12">C. elegans</span><span class="c1">, which were microscopically larger than tardigrades, thus there were problems on injecting them without them sliding around. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">New slides were made where a coverslip was placed on top and adhesed to a slide to prevent tardigrades from slipping. These slides would be used for next injection trip. </span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/a/ae/Genspace-2016-Notebook-9_24_16%281%29.jpg/450px-Genspace-2016-Notebook-9_24_16%281%29.jpg" alt="">
		
		</p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.ia1zhj2tg1k3"><span>9/26/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">To add downstream of parts to PRI__X__1C3 and to start overnight of PRI_MAH(S)_1C3</span></p><ul class="c3 lst-kix_pksebcn0vcs-0 start"><li class="c2"><span class="c1">Linearization and restriction digest of PRI_1C3</span></li><li class="c2"><span class="c1">Linearization, DpnI, PCR clean up of LEA proteins (HDLEA1, HDLEA3, and AAvLEA1) </span></li><li class="c2"><span class="c1">Ran a gel for the digests </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The gel for each of the samples came out nice and all samples loaded have the correct number of base pairs. Only one out of the four samples is chosen to start an overnight. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.uvwj6fyymr72"><span>9/27/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Ligation of PRI__1C3 with PRI__X, bacterial transformation for pSB1C3 with PRI__ and mini prepped J04450 and PRI_MAHS_1C3</span></p><ul class="c3 lst-kix_c0nm8qd6j65s-0 start"><li class="c2"><span class="c1">Made glycerol stock for PRI_MAHS_1C3 and J04450+pSB1C3</span></li><li class="c2"><span class="c1">Qubit reading after the mini prep</span></li><li class="c2"><span class="c1">Transformation of PRI_MAHS with plasmid K880005 (“5”)</span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">The backbone is successfully ligated and the transformation of MAHS has already taken place and plates will be checked on tomorrow for colony check and Colony PCR. </span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.qv9rv6iik0nb"><span>9/28/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Ligation of PR1_HDLEA1, 3, AAvLEA1 with 1C3 backbone, Colony PCR of the transformed plates as well as starting overnights for those Colony re-suspensions. </span></p><ul class="c3 lst-kix_4hkktm7ed5zn-0 start"><li class="c2"><span class="c1">Ligation of the remaining LEA proteins and also bacterial transformation  </span></li><li class="c2"><span class="c1">Started overnights for those samples </span></li></ul><p class="c5 c7"><span class="c1"></span></p><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Performed ligations of the remaining LEA proteins and also started overnights for them. By tomorrow we can Colony PCR the colonies to check for the correct base pairs. </span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.jcxxjlvwckhn"><span>Week 14: September 29 - October 6</span></h1><p class="c5 c7"><span></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.gcbzuws27wse"><span>9/29/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Colony PCR of plates from previous day. Preparation of primers to fix PstI site on HDLEA2. </span></p><ul class="c3 lst-kix_jnly3ciin1ne-0 start"><li class="c2"><span class="c1">Colony PCR conducted to conclude on the correct part and size for each supposed sample.</span></li><li class="c2"><span class="c1">Gel analyzation of PCR products</span></li><li class="c2"><span class="c1">Overnights of colony suspensions. </span></li><li class="c2"><span class="c1">Viewed tardigrades under the microscope </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Colony suspensions were inoculated into LB and were placed into rotating incubator to be grown for miniprep. </span></p><p class="c5"><span class="c1">*HDLEA2 will not be submitted as a part to registry due to the time it will consume to fix the PstI site. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5">

		<img src="http://2016.igem.org/wiki/images/thumb/7/72/Genspace-2016-Notebook-10_5_16.jpg/450px-Genspace-2016-Notebook-10_5_16.jpg" alt="">

		</p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.e3l7dpcnyvga"><span>10/6/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Bacterial transformation of ligations. </span></p><ul class="c3 lst-kix_bcb57aq858wa-0 start"><li class="c2"><span class="c1">Overnights of colony suspensions did not grow. </span></li><li class="c2"><span class="c1">Ligations from 9/28/16 were used to redo bacterial transformation. </span></li><li class="c2"><span class="c1">Note: 5 microliters of ligation PCR product was used for transformation. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Due to the colony suspensions not surviving living in fridge for more than a few days, they did not grow in overnight for miniprep. Bacterial transformation to insert ligated parts into cells. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.3scb9ahcn865"><span>Week 15: October 7- October 13</span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.t4ml3gsrukfr"><span>10/7/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Colony PCR of colonies from 10/6/16. Counting plates from desiccation experiment with PR1_MAHS_1C3. </span></p><ul class="c3 lst-kix_js31bt906wji-0 start"><li class="c2"><span class="c1">Picked colonies from plates and made colony suspensions in molecular biology grade water. </span></li><li class="c2"><span class="c1">Colony PCR products were analyzed on a gel to conclude on correct part and size. </span></li><li class="c2"><span class="c1">Plates from desiccation experiment on PR1_MAHS_1C3 were counted and data was collected. </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Process to sequence LEA proteins is almost at end. Colony PCR of colonies from transformed plates were picked for overnights. Plates from desiccation experiment were counted. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.r5thmlnjtfmt"><span>10/8/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Miniprep of colony overnights. </span></p><ul class="c3 lst-kix_v6y3kr2jp7jk-0 start"><li class="c2"><span class="c1">Glycerol stocks were made of the overnights. Flash freezed in liquid nitrogen. </span></li><li class="c2"><span class="c1">Mini prepped the 6 overnights. </span></li><li class="c2"><span class="c1">QuBit analysis of plasmid DNA concentrations. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Mini prep of overnights were measured for DNA concentration. One sample was picked from each protein and will be amplified for higher concentration before sequencing. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.i00de4iod2x6"><span>10/10/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Prepare samples for sequencing. </span></p><ul class="c3 lst-kix_iefnipnm21sk-0 start"><li class="c2"><span class="c1">Each plasmid DNA sample was calculated for a volume to be added to tubes to be sent out for sequencing. </span></li><li class="c2"><span class="c1">Three samples were prepared along with one forward primer and one reverse primer. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Samples from plasmid miniprep were prepared for sequencing. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.mvliaip5zfy0"><span>10/11/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Desiccation experiment for PR1_MAHS_IC3 and PR1_HDLEA1_1C3, PR1_HDLEA_3, and PR1_AAvLEA1_1C3. </span></p><ul class="c3 lst-kix_vlhbpmiq5euh-0 start"><li class="c2"><span class="c1">Start of desiccation experiment for LEA proteins. </span></li><li class="c2"><span class="c1">There are four different plates of each protein with a reference. Each are plated in triplets where there are three plates for each hour: 0, 4, 8, and 24 hours. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">Start of desiccation experiment for other LEA proteins. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.kox4rro1txf"><span>10/13/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Counting plates from desiccation experiment and overnight of other LEA proteins for next desiccation experiment. </span></p><ul class="c3 lst-kix_dslfnt3wbg7t-0 start"><li class="c2"><span class="c1">Plates on the agar side were marked with a 1 square centimeter square. </span></li><li class="c2"><span class="c1">Overnight of PR1_HDLEA1_1C3 and PR1_HDLEA3_1C3, with PR1_IC3 as control. </span></li></ul><p class="c5"><span class="c1 c4">Results: </span><span class="c1">After counting the plates, it showed that there may have been a contamination during plating since there were inconsistent colonies and there was a very widespread lawn of colonies. </span></p><p class="c5 c7"><span class="c1"></span></p>
	</div>


	<div class="week column full_size">
		<h1 class="c0" id="h.iq6is5xnt8dy"><span>Week 16: October 18 </span></h1><p class="c5 c7"><span class="c1"></span></p><h3 class="c0" id="h.tjuh54w45vx9"><span>10/18/16</span></h3><p class="c5"><span class="c1 c4">Objective: </span><span class="c1">Formatting parts into BioBricks to be submitted to iGEM registry. Digestion of GFP gBlock. </span></p><ul class="c3 lst-kix_ylfwxj8jnsqk-0 start"><li class="c2"><span class="c1">A list was made to direct each well of the plates for each part to be submitted. </span></li><li class="c2"><span class="c1">A total of 9 parts will be physically submitted. </span></li><li class="c2"><span class="c1">8 of the parts are plasmid DNA that have been placed into wells to be evaporated of liquid form into a dry form. </span></li><li class="c2"><span class="c1">Digest of GFP for </span><span class="c1 c12">H. dujardini</span><span class="c1">. </span></li><li class="c2"><span class="c1">Ligation of backbone with insert. </span></li><li class="c2"><span class="c1">Bacterial transformation of ligated part. </span></li></ul><p class="c5"><span class="c1 c4">Results:</span><span class="c1"> Plates for BioBrick standard were opened to dry out the plasmid DNA of each part to be submitted. Format for each part was filled in. Digest of GFP for ligation into pSB1C3 backbone to be plasmid DNA to be submitted. </span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1"></span></p><p class="c5 c7"><span class="c1 c4"></span></p></div>
	</div>
	<div class="week column full_size"></div>
</html>

