<h2 class="c0"><span class="c8">Background</span></h2>

<p class="c0"><span>The plasmid pSB1C3 is a favored vector for iGEM projects, its high copy number making it a workhorse for gene expression.  However, different sources give different estimates of its copy number and no iGEM teams have independently verified these claims.  Genspace developed a qPCR assay to verify plasmid copy number (PCN) in </span><span class="c5">E. coli</span><span> and made the first measurements of pSB1C3.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>The pSB1XX plasmid series uses the pMB1 origin or replication.  A point mutation in the replication primer, RNA II, allows the PCN to exceed 100 copies per cell, unless replication is repressed by the </span><span class="c5">rop</span><span> protein (Lin-Chao et al. 1992).  Plasmids encoding the </span><span class="c5">rop</span><span> gene, such as pBR322, have a much lower copy number, typically closer to 18 copies per cell (</span><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://drive.google.com/file/d/0B7L2OR2SM-b0QVNjRDBPdEFBclk/view?usp%3Dsharing&amp;sa=D&amp;ust=1476920272252000&amp;usg=AFQjCNFtU0kiUZdEskSgBWK9OarEFXUiYQ">Lee et al 2005</a></span><span>).  Additionally, incubation temperature has been shown to alter the copy number of plasmids using this origin of replication.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>To minimize variability caused by loss of nucleic acid during purification, a lysis protocol was researched that would allow cell lysate to be used directly as the template for qPCR (</span><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://drive.google.com/file/d/0B7L2OR2SM-b0dFRMYnlmVW9XM1E/view?usp%3Dsharing&amp;sa=D&amp;ust=1476920272254000&amp;usg=AFQjCNGubsJiD7ltmNgXLmUaD_U_F9xYWA">Shatzkes et al 2014</a></span><span>).  A TaqMan® hydrolysis probe was designed to target the LacZ gene and LacZ (BBa_K909006) was cloned into pSB1C3.  Because a copy of LacZ exists on the </span><span class="c5">E. coli</span><span> chromosome, lysate generated from cells without the plasmid was used as a standard, while lysate from cells with the plasmid was expected to have PCN+1 copies of the target sequence.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>In addition to measuring plasmid copy number, it was decided that steady-state mRNA count would also be a valuable measurement to make.  While we did not have enough time to make any mRNA measurements, the assay was designed with the intention of being usable on a DNA or RNA template.  To support this flexibility, the Verso One-Step qRT-PCR kit was chosen for our assay.  The Verso kit comes with an RT enhancer that degrades dsDNA during the reverse transcription step, allowing users to skip DNase treatment of samples.  Used as directed, the kit works for qRT-PCR, while withholding the RT enhancer and skipping the RT stage allows the kit to function as a qPCR assay.  Because samples do not require DNase treatment, both qPCR and qRT-PCR can be run using the same sample.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>Video: How TaqMan Works</span></p><p class="c0"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3DfkUDu042xic&amp;sa=D&amp;ust=1476920272256000&amp;usg=AFQjCNFrASby-X8IPgbQQIR3I04LSZ6JFw">https://www.youtube.com/watch?v=fkUDu042xic</a></span></p><p class="c0 c2"><span></span></p>


<h2 class="c0"><span class="c8">Sample Preparation Procedure</span></h2>
<p class="c0"><span>Bacterial cultures for testing were grown overnight to saturation, and the OD</span><span class="c12">600</span><span> was measured using the following formula:</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>OD</span><span class="c12">600</span><span> of 1.0 = 8x10</span><span class="c1">8</span><span> cells/mL</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>When necessary, 10-fold dilutions were prepared to ensure optical density remained within the linear range of our spectrophotometer.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>200,000,000 cells were taken and pelleted, then resuspended in 1mL of CL Buffer for a final concentration of 1 million cells per 5 microliters.  Cells were lysed at room temperature for at least 5 minutes, then frozen at -20°C.  10-fold dilutions of lysate were made using CL buffer to reduce DNA concentration to a manageable amount for qPCR.  For the standard, 20,000,000,000 cells were pelleted and lysed, and 10-fold serial dilutions in CL buffer were prepared.</span></p><p class="c0 c2"><span></span></p>


<h2 class="c0"><span class="c8">qPCR Procedure</span></h2>
<p class="c0"><span>Reactions were set up using the chemistry described in the kit’s manual.  A master mix was created and 20uL were distributed to each reaction tube.  5uL of cell lysate (containing 10</span><span class="c1">5</span><span> cells for the test articles) were added to each reaction.  A 3-point standard was used consisting of 10</span><span class="c1">5</span><span>, 10</span><span class="c1">6</span><span>, and 10</span><span class="c1">7</span><span> lysed </span><span class="c5">E. coli</span><span> cells without the plasmid.  All samples were run in triplicate.  The reverse transcription step was skipped and reverse transcriptase enzyme was inactivated during the 95°C hot start and qPCR continued normally using DNA as a template.</span></p><p class="c0 c2"><span class="c3"></span></p><p class="c0"><span class="c3">Each reaction has the following chemistry:</span></p><p class="c0 c4"><span class="c3">12.5uL Verso qRT-PCR Mix</span></p><p class="c0 c4"><span class="c3">5.45uL Molecular Biology Grade Water</span></p><p class="c0 c4"><span class="c3">5.0uL Cell lysate</span></p><p class="c0 c4"><span class="c3">1.8uL TaqMan Primer/Probe Mix</span></p><p class="c0 c4"><span class="c3">0.25uL Verso Enzyme Mix</span></p><p class="c0 c2"><span class="c3"></span></p><ol class="c11 lst-kix_9c2vkm96i5w4-0 start" start="1"><li class="c0 c7"><span class="c3">To minimize reaction variability, make a master mix first by multiplying the above volumes (withholding lysate) by the number of reactions you plan to run, with an excess of 2-3 reactions&#39; worth.</span></li><li class="c0 c7"><span class="c3">Distribute 20uL of master mix into each reaction tube.  Because the edge effect impairs data quality in the outer tubes, avoid using them for anything other than a positive or negative control</span></li><li class="c0 c7"><span class="c3">Distribute 5uL of the appropriate lysate (standard or test) to each reaction tube.  For the negative control, use 5uL of CL Buffer.</span></li></ol><p class="c0 c2"><span class="c3"></span></p><p class="c0"><span class="c3 c8">qPCR Cycles</span></p><p class="c0"><span class="c3">Run qPCR with the following cycles:</span></p><p class="c0 c4"><span class="c3">95C for 15 minutes</span></p><p class="c0 c4"><span class="c3">40 cycles of:</span></p><p class="c0 c6"><span class="c3">95C for 15 seconds</span></p><p class="c0 c6"><span class="c3">60C for 1 minute</span></p><p class="c0 c2"><span class="c8"></span></p>




<h2 class="c0"><span class="c8">Results</span></h2>

<p class="c0"><span>A preliminary experiment was run using lysate from 1 million cells per reaction.  Eight replicates were run for each sample type (with or without the reporter).  After excluding outliers caused by edge effects, average C</span><span class="c12">T</span><span> values for six replicates for each sample were compared:</span></p><p class="c0 c2"><span></span></p><p class="c0">

<span>[image: pSB1C3 Preliminary PCN Analysis]</span>

</p><p class="c0 c2"><span></span></p><p class="c0 c13"><span>Test:                17.02</span></p><p class="c0 c13"><span>Control:        21.48</span></p><p class="c0"><span>Assuming a perfect doubling of product for each C</span><span class="c12">T</span><span>, this would indicate 22 times as many copies of the target sequence in the test samples compared to the control, or a PCN of ~21 copies per cell.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>While this result was an order of magnitude less than the expected value, the consistency among replicates of each sample encouraged us to move forward with absolute quantification experiments.</span></p><p class="c0 c2"><span></span></p><p class="c0">

<span>[image: pSB1C3 Absolute Quantification 1]</span>

</p><p class="c0"><span>pSB1C3 absolute quantification run #1</span></p><p class="c0"><span>Lysate from 1 million stationary phase cells harboring K909006-pSB1C3 was run against a 3-point standard of 10</span><span class="c1">6</span><span>, 10</span><span class="c1">7</span><span>, and 10</span><span class="c1">8</span><span> copies.</span></p><p class="c0 c2"><span></span></p><p class="c0 c2"><span></span></p><p class="c0">

<span>[image: pSB1C3 Absolute Quantification 2]</span>

</p><p class="c0"><span>pSB1C3 absolute quantification run #2</span></p><p class="c0"><span>Lysate from 100,000 mid-log phase cells harboring K909006-pSB1C3 was compared against a 3-point standard of 10</span><span class="c1">5</span><span>, 10</span><span class="c1">6</span><span>, and 10</span><span class="c1">7</span><span> copies.  Due to the reduced amplification efficiency of the 10</span><span class="c1">8</span><span>-copy standard in run 1, cell numbers were reduced 10-fold in all subsequent experiments.</span></p><p class="c0 c2"><span></span></p><p class="c0"><span>Note: The K909006-pSB1C3 harboring cells used for this run were lysed in mid-log phase, which may account for the reduced PCN.</span></p><p class="c0 c2"><span></span></p><p class="c0">

<span>[image: pSB1C3 Absolute Quantification 3]</span>

</p><p class="c0"><span>pSB1C3 absolute quantification run #3</span></p><p class="c0"><span>Lysate from 100,000 stationary phase cells harboring K909006-pSB1C3 was compared against a 3-point standard of 10</span><span class="c1">5</span><span>, 10</span><span class="c1">6</span><span>, and 10</span><span class="c1">7</span><span> copies.</span></p><p class="c0 c2"><span></span></p><p class="c0">

<span>[image: pSB1C3 Absolute Quantification 4]</span>

</p><p class="c0"><span>pSB1C3 absolute quantification run #4</span></p><p class="c0"><span>Lysate from 100,000 stationary phase cells harboring K909006-pSB1C3 was compared against a 2-point standard of 10</span><span class="c1">5</span><span> and</span><span> 10</span><span class="c1">6</span><span> 1.1x10</span><span class="c1">6</span><span> copies.  The 1.1x10</span><span class="c1">6</span><span>-copy standard was created using lysate from 10</span><span class="c1">5</span><span> cells as well as 10</span><span class="c1">6</span><span> copies of purified plasmid.  This point was created to test for variance in amplification efficiency of plasmid vs. genomic template.</span></p><p class="c0 c2"><span></span></p>


<h2 class="c0"><span class="c8">Conclusions</span></h2>

<p class="c0"><span>Due to time constraints, only a few qPCR runs could be completed prior to the Wiki freeze, so this data is still preliminary.</span></p>